docs/content/en/docs-v1.0.x/installation/install-piped/installing-on-single-machine.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Installing on a single machine"
+linkTitle: "Installing on a single machine"
+weight: 5
+description: >
+  This page describes how to install a Piped on a single machine.
+---
+
+
+## Prerequisites
+
+### A registered `piped`
+
+- Make sure your `piped` is registered in the Control Plane and that you have its **PIPED_ID** and **PIPED_KEY**.  
+- If not, follow the guide to [register a new `Piped`](../../../user-guide/managing-controlplane/registering-a-piped/).
+
+### SSH Key for Git Repositories
+
+- If your Git repositories are private, `piped` requires a private SSH key to access those repositories.
+- Please checkout [this documentation](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) for how to generate a new SSH key pair. Then add the public key to your repositories. (If you are using GitHub, you can add it to Deploy Keys at the repository's Settings page.)
+
+If your Git repositories are private, `piped` needs an SSH key to access them.
+
+- Generate a new SSH key pair by following [GitHub’s guide to generating an SSH Key](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).  
+
+>Note:
+>If you are using GitHub, add the **public key** as a deploy key to your repositories.
+
+### Install the `piped` V1 binary
+
+Download the latest `piped` V1. See the [latest releases](https://github.com/pipe-cd/pipecd/releases) and find out the right binary for your machine.
+
+## Installation
+
+### Preparing the `Piped` configuaration file
+
+Plugins are external binaries that have to be referenced in the piped configuration file. There are no plugins set by default.
+
+An example of the a piped V1 configuration file using the Kubernetes plugin:
+
+```yaml
+apiVersion: pipecd.dev/v1beta1
+kind: Piped
+spec:
+  projectID: dev
+  pipedID: {PIPED_ID}
+  pipedKeyData: {PIPED_KEY}
+  apiAddress: {CONTROL_PLANE_API_ADDRESS} 
+  repositories:
+    - repoId: {REPO_ID_OR_NAME}
+      remote: git@github.com:{GIT_ORG}/{GIT_REPO}.git
+      branch: {GIT_BRANCH}
+  syncInterval: 1m
+  plugins:
+    - name: kubernetes
```

Comment:
Should use example plugin to keep this config common 👀 
https://github.com/pipe-cd/community-plugins/releases/tag/plugins%2Fexample-stage%2Fv0.1.0

How to use example plugin:
https://github.com/pipe-cd/community-plugins/tree/main/plugins/example-stage